.github/workflows/release_prep.yml

```diff
@@ -0,0 +1,67 @@
+name: release_prep
+
+on:
+  workflow_dispatch:
+    inputs:
+      module_version:
+        required: true
+
+jobs:
+  release_prep:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Extract branch name
+      run: echo ::set-env name=GITHUB_BRANCH::$(echo ${GITHUB_REF#refs/heads/})
+
+    - name: Set up Ruby
+      uses: ruby/setup-ruby@v1
+      with:
+        ruby-version: 2.5.3
+
+    - name: Update Rubygems
+      run: gem update --system 3.1.0
+
+    - name: Clone repository
+      uses: actions/checkout@v2
+      with:
+        ref: ${{ env.GITHUB_BRANCH }}
+        # Needed to fetch all the tags
+        fetch-depth: 0
+
+    - name: Generate the release prep commit
+      run: |
+        git checkout -b release_prep
+        bundle install --with release_prep
+        pdk release prep --version=${{ github.event.inputs.module_version }} --skip-changelog
+        git add .
+        git -c user.name=${{ github.actor }} -c user.email=${{ github.actor }}@users.noreply.github.com commit -m "Release prep for ${{ github.event.inputs.module_version }}"
+        git push --set-upstream origin release_prep --force
+
+    - name: Get most recent tag
+      id: most_recent_tag
+      uses: "WyriHaximus/github-action-get-previous-tag@master"
+      # An error in this step likely means that a most recent tag doesn't exist.
+      # This just means that our job corresponds to the first release, so continuing
+      # on error is OK.
+      continue-on-error: true
+
+    - name: Set PR_NOTE environment variable
+      run: |
+        if [ -z '${{ steps.most_recent_tag.outputs.tag  }}' ]; then
+          pr_note=''
+        else
+          pr_note='**Note:** You can use https://github.com/${{ github.repository }}/compare/${{ steps.most_recent_tag.outputs.tag  }}...${{ env.GITHUB_BRANCH }} to see all the new commits that have landed since the previous release.'
```

Comment:
Here's an example of a PR with this note -- https://github.com/ekinanp/test_release_automation/pull/16